docs/internals/howto-release-django.txt

```diff
@@ -2,9 +2,8 @@
 How is Django Formed?
 =====================
 
-This document explains how to release Django. If you're unlucky enough to
-be driving a release, you should follow these instructions to get the
-package out.
+This document explains how to release Django. If you're lucky enough to be
```

Comment:
Seems to me we should either take luck out of it completely or leave the sarcasm as is. :)